1.1.c

```diff
@@ -0,0 +1,100 @@
+#include <string.h>
+#include <stdio.h>
+
+#define binarySize 256

```

Comment:
```suggestion
#define BINARY_SIZE 256
```
Писать названия define констант нужно капсом (-0.25)